### PR TITLE
Feat: Add entities and repository for comment and vote

### PR DESCRIPTION
**Pull request by benjaminkostiuk**
## Overview

Implements entity and repository classes for `Vote` and `Comment` objects.

This change is a
- [ ] Bug fix
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that will change exisiting functionality)
- [ ] Documentation update

## Description

- Update boostrap schema to clarify naming and added `EDITED` column for comments.
- Updated MySQL schema to also create the comment and vote tables.
- Implement `Vote` and `Comment` entity classes along with required Enums.
- Implemented JPA repositories for both Vote and Comments.

## Motivation/Links

This implements the start of allowing users to vote and comment on test cases, comments and test suites.

## How was this tested?

Service compiles correctly, automated tests will follow after service and api classes are implemented.

## Todos
- [x] Ensure unit tests pass
- [ ] Update documentation for changes (if necessary)

### CODE REVIEW

File: src/main/java/com/unityTest/courseManagement/entity/Comment.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	// Type of source being commented on
+	@Enumerated(EnumType.STRING)
+	@NotNull
+	@Column(name = "SOURCE_TYPE")
+	private SourceType sourceType;
+
+	// Id of source item being commented on
+	@NotNull
+	@Column(name = "ITEM_ID")
+	private int itemId;
```
Comment (OZoneGuy) [inline src/main/java/com/unityTest/courseManagement/entity/Comment.java:42]: `itemId` is a bit ambiguous, why not:
```suggestion
	@NotNull
	@Column(name = "SOURCE_ID")
	private int sourceId;
```

or `parentId`. But I feel that `itemId` can be ambiguous if you are not familiar with the ER.

File: src/main/java/com/unityTest/courseManagement/entity/Vote.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	// Type of source being voted on
+	@Enumerated(EnumType.STRING)
+	@NotNull
+	@Column(name = "SOURCE_TYPE")
+	private SourceType sourceType;
+
+	// Id of source item being voted on
+	@NotNull
+	@Column(name = "ITEM_ID")
+	private int itemId;
```
Comment (OZoneGuy) [inline src/main/java/com/unityTest/courseManagement/entity/Vote.java:42]: See comment on `Comment.java`

**Review by OZoneGuy — CHANGES_REQUESTED** [review]
It seems like a straightforward PR. But I think that `itemId` is a bit ambiguous of a name.

File: src/main/java/com/unityTest/courseManagement/entity/Comment.java (hunk 1 above)
Comment (benjaminkostiuk) [reply src/main/java/com/unityTest/courseManagement/entity/Comment.java:42]: Agreed. `itemId` is a bit ambigous, but I'd like to keep the idea that it's the item's id not the source's. How about `sourceItemId`?

**Review by OZoneGuy — APPROVED** [review]

**Pull request merged**